src/MIDebugEngine/Engine.Impl/Variables.cs

```diff
@@ -400,10 +402,15 @@ public void AsyncError(IDebugEventCallback2 pExprCallback, IDebugProperty2 error
         }
 
         public void SyncEval(enum_EVALFLAGS dwFlags = 0)
+        {
+            SyncEval(DAPEvalFlags.NONE, dwFlags);
+        }
+
+        public void SyncEval(DAPEvalFlags dwDAPFlags, enum_EVALFLAGS dwFlags = 0)
         {
             Task eval = Task.Run(async () =>
             {
-                await Eval(dwFlags);
+                await Eval(dwFlags, dwDAPFlags);
```

Comment:
per my above comment: this is where it gets confusing. Eval takes dwFlags, DAP flags but SyncEval takes dwDAPFlags, dwflasg. I would change this method to support dapFlags.NONE as a default and keep consistent ordering.